addons/popochiu/editor/popups/create_prop/create_prop.gd

```diff
@@ -122,8 +125,7 @@ func _create() -> void:
 	# Abrir las propiedades de la prop creada en el Inspector
 	_main_dock.fs.scan()
 	await get_tree().create_timer(0.1).timeout
-	_main_dock.ei.edit_node(prop)
-	
+	_main_dock.ei.edit_node(prop_instance)
```

Comment:
Does this focus the scene tree on the node?

Reply:
Yup.

Reply:
TIL how to do this and it was something I wanted to do in the importer :D Thanks!